doc/user-guide/password.md

```diff
@@ -21,6 +21,77 @@ To enable it, simply turn the option on and set a user and a password:
 Your password isn't visible after deployment so write it down!
 ```
 
+## Basic Password Protection with the CLI
+
+Ploomber Cloud allows you to add authentication for specific features of your application using the CLI. This provides an additional layer of security for sensitive parts of your project.
+
+### Adding Authentication
```

Comment:
since analytics is only reserved to customers in the teams plan, it's not applicable to many customers.

show the examples for adding auth to the main app and then in a final section you can discuss adding/removing auth to the analytics

Reply:
> since analytics is only reserved to customers in the teams plan, it's not applicable to many customers.
> 
> show the examples for adding auth to the main app and then in a final section you can discuss adding/removing auth to the analytics

[showed examples for main app](https://github.com/ploomber/doc/pull/283/files#diff-ee5b667893ae9f81da782a6407f629f366501a0e2688f6ba2477c70cd8963a5cR28)